scripts/util.js

```diff
@@ -0,0 +1,41 @@
+const tmp = require("tmp");
+const ts = require("typescript");
+const path = require("path");
+const fs = require("fs-extra");
+
+tmp.setGracefulCleanup(); // cleans tmp file on process exit
+exports.corvidDir = path.join(__dirname, "../");
```

Comment:
why use the `corvidDir` as a tmp dir? Could it just as well have been any other dir? If so, better to get the dir path dynamically as opposed to having it hardCoded

Reply:
This more part of the existing process than it has to do with the module generation.
The original process creates a temp config and assumes the config passed to it is under the project directory and adds it as a prefix.
Since my file are already tmp and I need to delete it anyway I choose not to change that

Reply:
OK, then I think I understand the logic of not making this change as part of this commit, however, I do think it should be changed

Reply:
Agree, and I'll take your advice and do it in a second commit